src/test/java/com/datadog/api/v1/client/api/AwsIntegrationApiTest.java

```diff
@@ -59,6 +59,11 @@
     private static AWSAccountAndLambdaRequest uniqueAWSAccountLambdaRequest = new AWSAccountAndLambdaRequest();
     private static AWSLogsServicesRequest uniqueAWSLogsServicesRequest = new AWSLogsServicesRequest();
 
+    @Override
+    public String getTracingEndpoint() {
+        return "integration-aws";
```

Comment:
can we use class name of the `api` property?

Reply:
I want to make these independent of specific language/client to be able to make the monitoring setup easier in the future.

Reply:
I really think this should just be a slug of tag name (class name - `ApiTest` suffix -> camel case to kebab case).

Reply:
we can override it for custom tests like `telemetry`.

Reply:
Let's be explicit rather than implicit in this case, as forgetting to provide a value for a class like `LogsPipelinesLifecycleTest` would mark this as `logs-pipelines-lifecycle`. We would have to explicitly notice this, because nothing would tell us (this is also why I created no default).

Reply:
we could catch this during review. I would say we should discourage custom test classes that do not match spec tags.